userland/libtock/rng.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
```

Comment:
Actually not sarcastic.

I meant that the documentation in this file is great and I highly approve of it. However, there isn't a good way to comment on an entire file, so I just picked the first line (clearly poorly)